### PR TITLE
CTCP-5582: Add reference data call to incident code, pull incident de…

### PR DESCRIPTION
**Pull request by Reece-Carruthers**
…scription, refactor incorrect message key naming and removeu unused imports

### CODE REVIEW

**Comment by Reece-Carruthers** [comment]
image

**Review by kenneth-rayner — APPROVED** [review]

**Comment by Reece-Carruthers** [comment]
image
image

File: app/utils/IncidentP5MessageHelper.scala [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+class IncidentP5MessageHelper(
+  referenceDataService: ReferenceDataService
+)(implicit
+  messages: Messages,
+  ec: ExecutionContext,
+  hc: HeaderCarrier
+) extends DeparturesP5MessageHelper {
+
+  def incidentCodeRow(incidentCode: String, prefix: String): Future[Option[SummaryListRow]] =
```
Comment (jackheslop96) [inline app/utils/IncidentP5MessageHelper.scala:34]: As far as I can see the prefix is one of

`departure.notification.incidents.incident.code.label = Incident code`

`departure.notification.incident.index.code = Incident code`

File: app/utils/IncidentP5MessageHelper.scala [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+        buildRowFromAnswer[String](
+          answer = Some(incidentCode.toString),
+          formatAnswer = formatAsText,
+          prefix = prefix,
+          id = None,
+          call = None
+        )
+    }
+
+  def incidentDescriptionRow(incidentText: String, prefix: String): Option[SummaryListRow] = buildRowFromAnswer[String](
```
Comment (jackheslop96) [inline app/utils/IncidentP5MessageHelper.scala:46]: As far as I can see the prefix is one of

`departure.notification.incidents.incident.description.label = Description`

`departure.notification.incident.index.description = Description`

Comment (Reece-Carruthers) [reply app/utils/IncidentP5MessageHelper.scala:46]: Yep, either one of those. Are you thinking a boolean would be better to switch between them? Or just merge them into one prefix?

File: app/utils/IncidentsDuringTransitP5Helper.scala [hunk 3]
```diff
@@ -128,29 +106,35 @@ class IncidentsDuringTransitP5Helper(
...
+          sectionTitle = messages("departure.notification.incidents.subheading.incident", incidentIndex.display),
           rows = Seq(
             incidentCode,
-            incidentDescriptionRow(incidentIndex)
+            incidentDescriptionRow(
+              incidentText = data.Consignment.Incident(incidentIndex.position).text,
+              prefix = "departure.notification.incidents.incident.description.label"
+            )
           ).flatten,
           isOpen = if (incidentIndex.position == 0) true else false,
```
Comment (jackheslop96) [inline app/utils/IncidentsDuringTransitP5Helper.scala:131]: Not part of this PR but this is just `isOpen = incidentIndex.position == 0`

**Comment by Reece-Carruthers** [comment]
image
image

**Comment by Reece-Carruthers** [comment]
rebuild

**Review by jackheslop96 — APPROVED** [review]

**Review by kevincoatsworth — APPROVED** [review]
